docs/src/main/asciidoc/security-basic-authentication.adoc

```diff
@@ -11,22 +11,20 @@ include::_attributes.adoc[]
 :topics: security,authentication,basic-authentication,http
 :extensions: io.quarkus:quarkus-vertx-http,io.quarkus:quarkus-elytron-security-jdbc,io.quarkus:quarkus-elytron-security-ldap,io.quarkus:quarkus-security-jpa-reactive
 
-HTTP Basic authentication is one of the least resource-demanding techniques that enforce access controls to web resources.
-You can secure your Quarkus application endpoints with HTTP Basic authentication.
-Quarkus provides a built-in authentication mechanism for Basic authentication.
+HTTP Basic authentication is a low-resource method for enforcing access controls to web resources.
```

Comment:
Michal, that sentence which as far as I recall you created awhile back is clearer, `low-resource` reads unusual, and may be confusing

Reply:
OK, will keep it as originally written :)